dist/images/Makefile

```diff
@@ -17,36 +17,51 @@ DOCKERFILE_ARCH =
 ifeq ($(ARCH),arm64)
         DOCKERFILE_ARCH=.arm64
 endif
-OVS_BRANCH ?= main
-OVN_BRANCH ?= main
+IMAGE ?= ovn-daemonset-fedora
+
+OVN_REPO ?= https://github.com/ovn-org/ovn
+OVN_GITREF ?=
+ifeq ($(OVN_GITREF),)
+OVN_FROM := koji
+else
+OVN_FROM := source
+OVN_GITSHA := $(shell git ls-remote "${OVN_REPO}" "${OVN_GITREF}" | sort -k2  -V  |tail -1 | awk '{ print $$1 }')
+endif
+
+
 OCI_BIN ?= docker
 
 # The image of ovnkube/ovn-daemonset-ubuntu should be multi-arched before using it on arm64
-ubuntu: bld
+ubuntu-image: bld
```

Comment:
Hi @qinqon @jcaamano. This change needs to be reflected in the documentation.

Please take a look at https://github.com/ovn-kubernetes/ovn-kubernetes/pull/5136 and let's get that and anything else that needs updated addressed.